Tools/ApplicationDebugger/array-transform/README.md

```diff
@@ -133,6 +133,29 @@ Perform the following steps:
     $ make clean
     ```
 
+By default, CMake configures the build for Just-in-Time (JIT)
+compilation of the kernel.  However, it also offers an option for
+*Ahead-of-Time* (AoT) compilation.  To compile the kernel
+ahead-of-time for a specific device, set the `DPCPP_COMPILE_TARGET`
+option to the desired device during configuration.  For CPU, use the
+`cpu` value; for FPGA-emu, use the `fpga-emu` value.  Other values are
```

Comment:
To the best of my knowledge, FPGA *only* supports AoT compilation -- if you are discussing AoT, I think you need to mention this.

Nevermind, I was mistaken. FPGA emulator supports both JiT and AoT, although in practice it's extremely rare to use JiT. I withdraw my concern.